controllers/replace_failed_process_groups_test.go

```diff
@@ -911,6 +911,18 @@ var _ = Describe("replace_failed_process_groups", func() {
 				})
 			})
 		})
+
+		Context("with maintenance mode enabled", func() {
+			BeforeEach(func() {
+				adminClient, err := mock.NewMockAdminClientUncast(cluster, k8sClient)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(adminClient.SetMaintenanceZone("operator-test-1-storage-2", 0)).NotTo(HaveOccurred())
+			})
+
+			It("should not mark the process group for removal", func() {
+				Expect(getRemovedProcessGroupIDs(cluster)).To(Equal(BeEmpty()))
```

Comment:
```suggestion
				Expect(getRemovedProcessGroupIDs(cluster)).To(BeEmpty())
```

Reply:
Done.

Reply:
You have to remove the `Equal` it should be `To(BeEmpty())`, that's the reason the test is failing:

```text
 [FAILED] Expected
      <[]v1beta2.ProcessGroupID | len:0, cap:0>: []
  to equal
      <*matchers.BeEmptyMatcher | 0x37343b8>: {}
  In [It] at: /home/runner/work/fdb-kubernetes-operator/fdb-kubernetes-operator/controllers/replace_failed_process_groups_test.go:923 @ 06/29/23 22:28:21.368
------------------------------
••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••

Summarizing 1 Failure:
  [FAIL] replace_failed_process_groups with a process that has been missing for a long time with maintenance mode enabled [It] should not mark the process group for removal
  /home/runner/work/fdb-kubernetes-operator/fdb-kubernetes-operator/controllers/replace_failed_process_groups_test.go:923
```

Your current statement checks if `getRemovedProcessGroupIDs(cluster)` equals the `BeEmpty()` matcher.

Reply:
Sorry, it is addressed now.